packages/ng2-qgrid-app/tsconfig.json

```diff
@@ -15,7 +15,24 @@
     "lib": [
       "esnext",
       "dom"
-    ]
+    ],
+		"paths": {
```

Comment:
it should not be like that please. see "references" section, if you add @angular/ here, something wrong